config/prow/external-secrets/README.md

```diff
@@ -0,0 +1,14 @@
+## Steps for fetching secret from Secret Manager to k8s cluster
+
+### 1. Deploying external Secrets Operator on cluster
+helm repo add external-secrets https://charts.external-secrets.io
+helm install external-secrets external-secrets/external-secrets -n prow
+
+### 2. Creating ibm-secret in cluster for connecting to IBM Cloud Secrets Manager
+`kubectl create secret generic ibm-secret --from-literal=apiKey='API_KEY_VALUE'`
```

Comment:
can we use any existing k8s secret or want to create a new one?

Reply:
This is no existing secret with API_KEY value in prow namespace.
Have added the namespace `prow` to this line now.

Reply:
As this secret is needed in all namespaces using the public certificate, have changed to  `-n <namespace>`